core-services/prow/02_config/_plugins.yaml

```diff
@@ -5873,6 +5873,7 @@ plugins:
   - wip
   - yuks
   openshift-priv/debug-network:
+  - approve
```

Comment:
Not `openshift-priv`

Reply:
hehe, i thought openshift-priv was something like 'openshift privileges', not an org, thanks.
Done now